### PR TITLE
pages

### PR DESCRIPTION
**Pull request by ragini07**
added pages

### CODE REVIEW

File: src/Components/Header/Header.js [hunk 1]
```diff
@@ -1,9 +1,107 @@
...
+              <img src={smLogo}></img>
+            </div>
+          </div>
+
+          {/* <input
+    
+          className="text-gray-700 border h-8 mt-3 border-gray-300 flex items-center align-items p-1 w-1/4 flex items-center align-items rounded focus:outline-none focus:ring-1 focus:ring-purple-500 lg:w-2/5 sm:hidden"
+          type="text"
+          placeholder="type to search"
+        /> */}
```
Comment (znjs) [inline src/Components/Header/Header.js:23]: can remove the commented code

**Review by znjs — APPROVED** [review]
- Clean UI
- Good PR
- Good naming conventions used

**Review by shraddha15-creator — APPROVED** [review]
UI Looks good
Good folder structure

File: src/Components/PrivateRoute/PrivateRoute.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { navigate, useNavigate, Route, Navigate } from "react-router-dom";
+
+
+function PrivateRoute({ children }) {
+  const navigate = useNavigate();
```
Comment (jeetsdev) [inline src/Components/PrivateRoute/PrivateRoute.js:6]: I don't think so you are using navigate here, so you can remove it.

**Review by JowelTisso — APPROVED** [review]
* Good work with tailwind 
* Modular code and clean code
* Remember to remove used code
* Rest looks good

**Review by jeetsdev — APPROVED** [review]
Everything looks good to merge.

- Clean UI
- Good folder structure

File: src/Components/PrivateRoute/PrivateRoute.js (hunk 2 above)
Comment (ragini07) [reply src/Components/PrivateRoute/PrivateRoute.js:6]: I will use it in next pr for auth

**Pull request merged**